cli/jobs/dataprep/dask/dask-job.yml

```diff
@@ -0,0 +1,33 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/commandJob.schema.json
+code: 
+  local_path: src
+
+# This is the command that will start up the dask cluster and run the script `batch.py` with the following parameters
+# for an interactive session, just remove the --script. That will just start the cluster
+# and mount the dataset (which will be found under './AZURE_ML_INPUT0')
+command: >-
+  python startDask.py
+  --script batch.py 
+  --nyc_taxi_dataset {inputs.nyc_taxi_dataset} 
+  
+environment: 
+  conda_file: file:dask-conda.yaml
```

Comment:
this is being deprecated, can we use Docker context or a curated environment instead?